### ISSUE TITLE
Notification on card due date.

### ISSUE THREAD

**Issue by Mathijs0** [wekan/wekan#5323]
Notification reminder (webhook) on card/task due date, when no end date.
Maybe with an offset setting per board. Like 0 on the day of and -1 day before.

Just an idea

**Comment by Veka85** [comment]
Yes, we really need a notification when card is due (and ideally one also day before)...
